README.md

```diff
@@ -88,35 +88,20 @@ sudo apt-get update
 sudo apt-get install git build-essential binutils-mips-linux-gnu python3
 ```
 
-#### 2. Download and set up qemu-irix (optional)
-
-Note: We are currently testing a recompiled version of the compiler that does not require qemu-irix.
-This step is optional and allows you to build with qemu-irix and the original compiler by adding `ORIG_COMPILER=1` to the `make` command in step 6, for example if you experience issues with the recompiled version.
-
-Download qemu-irix from the Releases section in the repository. Place it at a location of your choosing.
-
-Open up your .bashrc file (~/.bashrc), scroll to the bottom, and add the following, replacing the paths as necessary:
-
-```text
-export QEMU_IRIX="path/to/your/qemu-irix"
-```
-
-Save and close/reopen your terminal window.
-
-#### 3. Clone the repository
+#### 2. Clone the repository
```

Comment:
Since this changes the step numbers, I think there are references to steps that need to be updated in the rest of the README (in the Docker instructions iirc but there might be more)

Reply:
Good catch!